### ISSUE TITLE
[NTR]: ZW sex-determination system 

### ISSUE THREAD

**Issue by mikewhitelab** [Tree-of-Sex/tos-ontology#36]
### In which category is your new term request?

- [ ] new_definition
- [X] existing_definition

### Preferred term label

ZW sex-determination

### Textual definition

A chromosomal system that determines the sex of offspring in birds, some fish and crustaceans such as the giant river prawn, some insects (including butterflies and moths), and some reptiles, including Komodo dragons. In this system, female has a pair of dissimilar ZW chromosomes and male has two similar ZZ chromosomes.
http://purl.obolibrary.org/obo/GSSO_000119

### Suggested parent term

sex chromosomes
sex determination

### Additional context

The existing definition provides a list of examples, but this is not comprehensive. Could be misleading. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by enemeshazi** [comment]
Yes, this is an incomplete list of examples, e.g. ZW/ZZ systems occur in amphibians as well. But I think we shouldn't list taxa in ontology definitions in general, because the system/phenomenon etc. may be recognized later in further taxa as well, so even if the list seems correct now, it might become incomplete and misleading later. I think we should define ZW and XY sex-determination systems in a uniform style. My suggestion for XY sex-determination system is at https://github.com/Tree-of-Sex/ToS-Ontology/issues/19 (but it might also need some refinement)

**Comment by anyatorgasheva** [comment]
I also agree that we should not use examples in the general definitions. I tried to align the definition with mentioned above XY system. - ZW sex-determination system is a chromosomal system in gonochoristic organisms where females are the heterogametic sex, typically having two dissimilar ZW chromosomes, and males are the homogametic sex, typically having two similar ZZ chromosomes. Complex ZW systems can also occur, where multiple chromosome pairs contribute to sex determination.

**Comment by ASEMBER2** [comment]
I mostly agree with the comments above but I would not mention complex ZW systems here as it is confusing. Complex systems may include not only two different ZW systems but also co-segregating XY and ZW systems either i) each system in different individuals is the same population or in different populations, or ii) these two systems co-segregating in the same individual - and here is the question if it is a polymorphism in few individuals or a feature of the entire population. Complex systems are quite "complex", indeed. Another term is "multiple sex chromosomes" which needs to be distinguished from "complex systems".

Also "dissimilar ZW chromosomes" needs to be carefully refined to better encompass homomorphic systems.

My suggestion for a definition for ZW system is (bearing in mind the above comments):

ZW sex-determination system is a chromosomal system in gonochoristic organisms where females are the heterogametic sex, typically having two distinct sex chromosomes Z and W, with varying degree of genetic content differentiation, and males (the homogametic sex) having two copies of a Z chromosome. 

Eventually to add the note:
More than one distinct Z and/or W chromosome may be also present in the particular karyotype/genome (multiple sex chromosome system). ZW sex chromosome system may also co-segregate with another sex chromosome system (ZW or XY) within the same individual/population/species (complex sex chromosome systems.

**Comment by KamilSJaron** [comment]
I like the refined definitions by Alex

**Comment by anyatorgasheva** [comment]
I agree with the refined definition

**Comment by ValenzuelaLab** [comment]
I favor this modified definition:

ZW sex-determination system is a chromosomal system of genotypic sex determination in gonochoristic organisms where females are the heterogametic sex, carrying two distinct sex chromosomes (labeled Z and W) that differ in genetic content (and may differ morphologically or not from one another), whereas males are the homogametic sex and carry two copies of the Z chromosome.

I would suggest leaving the notes on complex and multiple sex chromosome systems to a separate definition. 
We should also have definitions for male heterogamety and female heterogamety, which would encompass complex sex chromosome systems.

**Comment by ASEMBER2** [comment]
I agree with this definition:

ZW sex-determination system is a chromosomal system of genotypic sex determination in gonochoristic organisms where females are the heterogametic sex, carrying two distinct sex chromosomes (labeled Z and W) that differ in genetic content (and may differ morphologically or not from one another), whereas males are the homogametic sex and carry two copies of the Z chromosome.

**Comment by mrovatsos** [comment]
Modifying the above definition:


ZW sex-determination system is a chromosomal system of genotypic sex determination, where females are the heterogametic sex, carrying two distinct sex chromosomes (labeled Z and W) that differ in genetic content, whereas males are the homogametic sex and carry two copies of the Z chromosome.

similar definition should be used for XY sex-determination:
XY sex-determination system is a chromosomal system of genotypic sex determination, where males are the heterogametic sex, carrying two distinct sex chromosomes (labeled X and Y) that differ in genetic content, whereas females are the homogametic sex and carry two copies of the X chromosome.

**Comment by ASEMBER2** [comment]
I agree. The point of the modification by Michail was (after discussions here) that also hermaphrodites may have sex chromosomes (so, a note about gonochoristic organisms was excluded).

**Comment by PaulYannJay** [comment]
As with the XY system, I propose a simplification:

Chromosomal system of genotypic sex determination, where females carry two distinct sex chromosomes, labeled Z and W, whereas females carry two copies of the Z chromosome.

**Comment by ValenzuelaLab** [comment]
I agree with this simplified definition:

Chromosomal system of genotypic sex determination, where females carry two distinct sex chromosomes, labeled Z and W, whereas females carry two copies of the Z chromosome.
